parsl/providers/kubernetes/kube.py

```diff
@@ -161,9 +159,10 @@ def cancel(self, job_ids):
         for job in job_ids:
             logger.debug("Terminating job/proc_id: {0}".format(job))
             # Here we are assuming that for local, the job_ids are the process id's
-            self._delete_deployment(job)
```

Comment:
Maybe remove _delete_deployment from the source file?

Reply:
fixed